### PR TITLE
Fix some typo and grammar

### PR DESCRIPTION
**Pull request by 928234269**
**What this PR does / why we need it**:
This PR corrected grammar in docs/charts.md and fixes a simple typo in  pkg/storage/storage.go, pkg/kube/client.go

**Special notes for your reviewer**:
**If applicable**:
- [x] this PR contains documentation

### CODE REVIEW

File: docs/charts.md [hunk 1]
```diff
@@ -479,7 +479,7 @@ Furthermore, A is dependent on chart B that creates objects
 - replicaset "B-ReplicaSet"
 - service "B-Service"
 
-After installation/upgrade of chart A a single Helm release is created/modified. The release will
```
Comment (bacongobbler) [inline docs/charts.md]: this is not a typo. Perhaps the grammar should be corrected instead, adding a comma after "chart A". e.g.

> After installation/upgrade of chart A, a single Helm release is created/modified.

**Review by bacongobbler — CHANGES_REQUESTED** [review]

**Comment by bacongobbler** [comment]
see https://github.com/helm/helm/pull/6223/checks?check_run_id=193168428

File: docs/charts.md (hunk 1 above)
Comment (928234269) [reply docs/charts.md]: I have corrected the grammar of this part.

**Comment by 928234269** [comment]
@bacongobbler  This pr failed in the test, the strange thing is that I only modified the documentation and comments.Can you give me some help? https://circleci.com/gh/helm/helm/9116?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link

**Comment by mattfarina** [comment]
If you see `/usr/local/go/pkg/tool/linux_amd64/link: signal: killed` in the logs of a failed test run it is due to circleci and memory. The current helm tests sometimes fail due to running out of memory. This is fixed in helm v3 but an issue sometimes here. We have to re-run tests so they pass.

**Comment by 928234269** [comment]
> 如果您`/usr/local/go/pkg/tool/linux_amd64/link: signal: killed`在测试失败的日志中看到它是由于circleci和内存。由于内存不足，当前的helm测试有时会失败。这在helm v3中已修复，但有时会出现问题。我们必须重新运行测试，以便通过。

**Comment by 928234269** [comment]
> If you see `/usr/local/go/pkg/tool/linux_amd64/link: signal: killed` in the logs of a failed test run it is due to circleci and memory. The current helm tests sometimes fail due to running out of memory. This is fixed in helm v3 but an issue sometimes here. We have to re-run tests so they pass.

What do I need to do to retest?

**Comment by 928234269** [comment]
@bacongobbler @mattfarina I created a new pr and closed the pr because I can't retest it. This new pr is here https://github.com/helm/helm/pull/6236.

**Pull request closed without merging**